python/adios2/stream.py

```diff
@@ -315,7 +315,7 @@ def _(self, name, content, shape=[], start=[], count=[], operations=None):
         if shape != [] and not variable.single_value():
             variable.set_shape(shape)
 
-        if start != [] and count != []:
+        if start != [] or count != []:
```

Comment:
I thought start and count need to match in length

Reply:
For local arrays, start is empty, just like shape.